loader/util.go

```diff
@@ -73,3 +76,14 @@ func shortSha1(s string) string {
 func percent(a int64, b int64) string {
 	return fmt.Sprintf("%.2f %%", float64(a)/float64(b)*100)
 }
+
+func generateSchemaCreateFile(dir string, schema string) error {
+	file, err := os.Create(path.Join(dir, fmt.Sprintf("%s-schema-create.sql", schema)))
+	if err != nil {
+		return errors.Trace(err)
+	}
+	defer file.Close()
+
+	file.WriteString(fmt.Sprintf("CREATE DATABASE `%s`;\n", schema))
```

Comment:
Use `fmt.Fprintf`?

What if `schema` contains a `` ` ``?

Reply:
also need to check error returned from `fmt.Fprintf` or `file.WriteString`

Reply:
addressed in https://github.com/pingcap/dm/pull/186/commits/25659eee5dfb4a70f92960b721fe9d7fac819f15

Reply:
do you have any suggest when schema contains ```? @kennytm

Reply:
There should be a function somewhere which replaces `` ` `` with ``` `` ```

Reply:
done in https://github.com/pingcap/dm/pull/186/commits/d87c9591b3e8676567cb2997f23cde56b9918707
PTAL again